spec/javascript/packages/document-capture-polling/index-spec.js

```diff
@@ -44,10 +44,6 @@ describe('DocumentCapturePolling', () => {
     subject.bind();
   });
 
-  afterEach(() => {
-    subject.bindPromptOnNavigate(false);
-  });
-
```

Comment:
While it was added in #8512, this feels like a clean-up task that ought to have been included all along, so I'd recommend keeping.

Reply:
I'll re-add it back in.